CRM/Contribute/BAO/Contribution.php

```diff
@@ -5559,6 +5502,72 @@ protected static function getContributionPaymentLinks($id, $balance, $contributi
     return $actionLinks;
   }
 
+  /**
+   * @param array $contactIDs
+   *
+   * @return string
+   */
+  public static function getAnnualQuery($contactIDs) {
+    $contactIDs = implode(',', $contactIDs);
+    $config = CRM_Core_Config::singleton();
+    $currentMonth = date('m');
+    $currentDay = date('d');
+    if ((int ) $config->fiscalYearStart['M'] > $currentMonth ||
+      ((int ) $config->fiscalYearStart['M'] == $currentMonth &&
+        (int ) $config->fiscalYearStart['d'] > $currentDay
```

Comment:
Please bracket this - I know the order of execution by it can be confusing.

Reply:
I changed the line breaks which I think makes it clear